src/current/v23.2/physical-cluster-replication-monitoring.md

```diff
@@ -0,0 +1,131 @@
+summary: Monitor and observe replication streams between a primary and standby cluster.
+toc: true
+docs_area: manage
+---
+
+{{site.data.alerts.callout_info}}
+{% include feature-phases/preview.md %}
+{{site.data.alerts.end}}
+
+{% include_cached new-in.html version="v23.2" %} You can monitor a physical cluster replication stream using:
+
+- [`SHOW VIRTUAL CLUSTER ... WITH REPLICATION STATUS`](#sql-shell) in the SQL shell.
+- The [Physical Replication dashboard](#db-console) on the DB Console.
+- [Prometheus and Alertmanager](#prometheus) to track and alert on replication metrics.
+
+{% comment %}Add data verification on standby here?{% endcomment %}
+
+
+{% comment %}This page describes the available monitoring for physical cluster replication, for information on how replication works, refer to:{% endcomment %}
+
+{% include {{ page.version.version }}/physical-replication/reference-links-replication.md %}
+
+## SQL Shell
+
+In the standby cluster's SQL shell, you can query `SHOW VIRTUAL CLUSTER ... WITH REPLICATION STATUS` for detail on status and timestamps for planning [cutover]({% link {{ page.version.version }}/cutover-replication.md %}):
+
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+ SHOW VIRTUAL CLUSTER standbyapplication WITH REPLICATION STATUS;
+~~~
+
+Refer to the [Responses](#responses) for a description of each field.
+
+{% include_cached copy-clipboard.html %}
+~~~
+id |        name        |     data_state     | service_mode | source_tenant_name |                                                     source_cluster_uri                                               | replication_job_id |        replicated_time        |         retained_time         | cutover_time
+---+--------------------+--------------------+--------------+--------------------+----------------------------------------------------------------------------------------------------------------------+--------------------+-------------------------------+-------------------------------+---------------
+3  | standbyapplication | replicating        | none         | application        | postgresql://{user}:{password}@{hostname}:26257/?options=-ccluster%3Dsystem&sslmode=verify-full&sslrootcert=redacted | 899090689449132033 | 2023-09-11 22:29:35.085548+00 | 2023-09-11 16:51:43.612846+00 |     NULL
+(1 row)
+~~~
+
+### Responses
+
+Field    | Response
+---------+----------
+`id` | The ID of the virtual cluster.
+`name` | The name of the standby (destination) virtual cluster.
+`data_state` | The state of the data on the virtual cluster. This can show one of the following: `initializing replication`, `ready`, `replicating`, `replication paused`, `replication pending cutover`, `replication cutting over`, `replication error`. Refer to [Data state](#data-state) for more detail on each response.
+`service_mode` | The service mode shows whether the virtual cluster is ready to accept SQL requests. This can show one of the following: `none`, `shared`.
```

Comment:
Are these all of the modes? And what does "shared" technically mean?

Reply:
I have added a sentence for `shared`. 

For `external`, do we want to include that here as a part of physical rep? If so, I'm not sure how that is defined?